Packs/McAfee_Advanced_Threat_Defense/Integrations/McAfee_Advanced_Threat_Defense/McAfee_Advanced_Threat_Defense.py

```diff
@@ -779,7 +779,7 @@ def main():
         # return detonate(3, args.url, args.timeout, args.format, args.threshold);
         # submit type for url-download is 3
 
-    except Exception, ex:
+    except Exception as ex:
```

Comment:
This is python 2 integration, not python 3

Reply:
This syntax works in BOTH.

Reply:
This syntax works in BOTH.